src/python/pants/option/option_util_test.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from unittest import TestCase
+
+from pants.option.option_util import flatten_shlexed_list
+
+
+class OptionUtilTest(TestCase):
+
+  def test_flatten_shlexed_list(self) -> None:
+    assert flatten_shlexed_list(["arg1", "arg2"]) == ("arg1", "arg2")
+    assert flatten_shlexed_list(["arg1 arg2"]) == ("arg1", "arg2")
+    assert flatten_shlexed_list(["arg1 arg2=foo", "--arg3"]) == ("arg1", "arg2=foo", "--arg3")
```

Comment:
Can you chuck in one more for quoted strings, something like:
```python
assert flatten_shlexed_list(["arg1 arg2='foo bar'", "--arg3"]) == ("arg1", "arg2=foo bar", "--arg3")
```

Reply:
Good call!